### PR TITLE
Fix quote and example indentation

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

File: docs/src/main/resources/jasper-reports-part.html [hunk 1]
```diff
@@ -95,7 +95,7 @@
 <p>One of the most important change from version 2 to 3 of Mapfish Print, at 
 least for users, 
 is the use of jasper template (jrxml) as basis of layout. The chapter 
-“General workflow for adapting existing templates”
+"General workflow for adapting existing templates"
```
Comment (sbrunner) [inline docs/src/main/resources/jasper-reports-part.html]: Why do you change the quotes?
The previous one will not be better?

Comment (Vampouille) [inline docs/src/main/resources/jasper-reports-part.html]: On the generated website, there is an encoding issue with the closing quote. You can see encoding issue here : 
http://mapfish.github.io/mapfish-print-doc/#/jasperReports
Others quote use regular quote character (ASCII quote) in this page.

Comment (Vampouille) [inline docs/src/main/resources/jasper-reports-part.html]: I think non-Ascii quote should be entered with html entities : 	&ldquo; and &rdquo;. 
Personnaly, I would prefer traditional ascii quote !

Comment (sbrunner) [inline docs/src/main/resources/jasper-reports-part.html]: Good point -> ok for me :-)

**Comment by sbrunner** [comment]
Thanks @Vampouille :-)

**Pull request merged**